api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -123,19 +130,21 @@ private void createMakePaymentMessage(PaymentCycle paymentCycle) {
         messageQueueClient.sendMessage(messagePayload, MessageType.MAKE_PAYMENT);
     }
 
-    private void handleLossOfQualifyingBenefitStatus(Claim claim) {
+    private void handleLossOfQualifyingBenefitStatus(Claim claim, List<LocalDate> dateOfBirthOfChildren) {
         updateClaimStatus(claim, ClaimStatus.PENDING_EXPIRY);
         determineEntitlementNotificationHandler.sendClaimNoLongerEligibleEmail(claim);
+        claimMessageSender.sendReportClaimMessage(claim, dateOfBirthOfChildren, UPDATED_FROM_ACTIVE_TO_PENDING_EXPIRY);
     }
 
-    private void handleNoLongerEligibleForSchemeAsNoChildrenAndNotPregnant(Claim claim) {
-        expireClaim(claim);
+    private void handleNoLongerEligibleForSchemeAsNoChildrenAndNotPregnant(Claim claim, List<LocalDate> dateOfBirthOfChildren) {
+        expireClaim(claim, dateOfBirthOfChildren);
         determineEntitlementNotificationHandler.sendNoChildrenOnFeedClaimNoLongerEligibleEmail(claim);
     }
 
-    private void expireClaim(Claim claim) {
+    private void expireClaim(Claim claim, List<LocalDate> dateOfBirthOfChildren) {
         updateClaimStatus(claim, ClaimStatus.EXPIRED);
         eventAuditor.auditExpiredClaim(claim);
+        claimMessageSender.sendReportClaimMessage(claim, dateOfBirthOfChildren, UPDATED_FROM_ACTIVE_TO_EXPIRED);
```

Comment:
can we be sure they are active currently? I think it's possible that they were pending expiry. (I think this would mean that they became eligible again this cycle, but it happened to coincide with their children turning 4)

Reply:
The method is currently only called if the claim is active, I'll rename the private method to expireActiveClaim which should prevent it being called when we handle the case of pending_expiry to expired.

Reply:
Went for renaming the private method to expireActiveClaim